schema/schema-solaris.json

```diff
@@ -0,0 +1,36 @@
+{
+    "solaris": {
+        "description": "Solaris platform-specific configurations",
+        "id": "https://opencontainers.org/schema/bundle/solaris",
+        "type": "object",
+        "properties": {
+            "milestone": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/milestone",
+                "type": "string"
+            },
+            "limitpriv": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/limitpriv",
+                "type": "string"
+            },
+            "max-shm-memory": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/max-shm-memory",
+                "type": "string"
+            },
+            "capped-cpu": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/capped-cpu",
+                "$ref": "defs.json#/definitions/mapStringString"
```

Comment:
This is accurate, but it could be more specific if you explicitly lists allowed keys.  That can happen with a [separate definition][1] or [inline][2].

[1]: https://github.com/opencontainers/runtime-spec/blob/e5e603f26fe8119b564e506d66e2b3426a68f25f/schema/schema-linux.json#L83-L86
[2]: https://github.com/opencontainers/runtime-spec/blob/e5e603f26fe8119b564e506d66e2b3426a68f25f/schema/schema-linux.json#L218-L243